sfa_dash/blueprints/aggregates.py

```diff
@@ -292,6 +310,7 @@ def set_template_args(self):
             'breadcrumb': self.breadcrumb_html(
                 self.get_breadcrumb_dict()),
             'aggregate': self.metadata,
+            'subnav': self.format_subnav(**self.get_subnav_kwargs())
```

Comment:
Passing dicts around like this makes it really hard to track down exactly what is happening. Why not pass the keywords here and get rid of ``get_subnav_kwargs`` completely?